.github/workflows/setup-pr.yml

```
@@ -0,0 +1,60 @@
+            echo "Could not fetch issue #$ISSUE_NUMBER, skipping."
+            exit 0
+          fi
+
+          echo "Issue title: $ISSUE_TITLE"
+
+          # Update the PR title
+          gh pr edit "$PR_NUMBER" --title "$ISSUE_TITLE"
+
+          # Update the PR body
+          gh pr edit "$PR_NUMBER" --body "Fixes #$ISSUE_NUMBER
+
```

actionlint (1.7.11)

[error] could not parse as YAML: did not find expected key

(syntax-check)